sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,220 @@
+
+            The manufacturing environment is required to support test reruns whenever possible:
+
+            - Verify that the RAW_UNLOCK process can be re-tried after failed or interrupted attempts.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_scrap
+      desc: '''Verify transition from any state to SCRAP lc_state.
+
+            Manufacturing tests should be able to move the device from any lc_state into SCRAP state.
+
+            For each manufacturing lc_state `s`:
+            - Pre-load OTP with `s` lc_state image.
+            - Bring device out of reset selecting the Life Cycle (LC) TAP interface using strap pins.
+            - Request transition to SCRAP state via LC TAP interface.
+            - Perform reset by togglig the POR pin.
+            - Check we are in SCRAP state via LC TAP interface.
+
+            The manufacturing environment is required to support test reruns whenever possible:
+
+            - Verify that the transition to SCRAP mode can be re-tried after failed or interrupted attempts.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_cp_yield_test
+      desc: '''Verify that it is possible to switch TAP interfaces while injecting external clock.
+
+            Verify that it is possible to access TAP interfaces while injecting an external clock.
+            This test case is valid only in test life cycle states.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Enable external clock injection.
+            - Connect to CPU JTAG and perform register read/writes to memory/registers.
+            - Switch to LC TAP by manipulating the strap pins.
+            - Verify that read/writes to lc_ctrl work as expected.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_cp_ast_test_execution
+      desc: '''Verify that the tester is able to load/exec test sequences into/from SRAM.
+
+            An expected pattern during manufacturing involves loading small program sequences into
+            SRAM, and use them to run higher level operations such as configuring OTP values in the
+            OTP hardware partition.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Load program into SRAM to execute OTP word write sequences.
+            - Trigger OTP writes to HW partition (e.g. device ID).
+            - Perform reset by toggling POR pin.
+            - Select CPU JTAG via strap pins.
+            - Verify that OTP write was successful.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_cp_device_info_flash_wr
```

Comment:
this is similar to #14486 and #14484 but those tests are defined to decouple them from other peripherals. We can build on those or just use this for #14486 as well.